docs/src/man/reshaping_and_pivoting.md

```diff
@@ -22,41 +95,187 @@ The stacked DataFrame that results includes all of the columns not specified to
 
 A third optional argument to `stack` represents the id columns that are repeated. This makes it easier to specify which variables you want included in the long format:
 
-```julia
-d = stack(iris, [:SepalLength, :SepalWidth], :Species)
+```jldoctest
+julia> using DataFrames, CSV
+
+julia> iris = CSV.read(joinpath(Pkg.dir("DataFrames"), "test/data/iris.csv"));
+
+julia> d = stack(iris, [:SepalLength, :SepalWidth], :Species);
+
+julia> head(d)
+6×3 DataFrames.DataFrame
+│ Row │ variable    │ value │ Species │
+├─────┼─────────────┼───────┼─────────┤
+│ 1   │ SepalLength │ 5.1   │ setosa  │
+│ 2   │ SepalLength │ 4.9   │ setosa  │
+│ 3   │ SepalLength │ 4.7   │ setosa  │
+│ 4   │ SepalLength │ 4.6   │ setosa  │
+│ 5   │ SepalLength │ 5.0   │ setosa  │
+│ 6   │ SepalLength │ 5.4   │ setosa  │
+
+julia> tail(d)
+6×3 DataFrames.DataFrame
+│ Row │ variable   │ value │ Species   │
+├─────┼────────────┼───────┼───────────┤
+│ 1   │ SepalWidth │ 3.3   │ virginica │
+│ 2   │ SepalWidth │ 3.0   │ virginica │
+│ 3   │ SepalWidth │ 2.5   │ virginica │
+│ 4   │ SepalWidth │ 3.0   │ virginica │
+│ 5   │ SepalWidth │ 3.4   │ virginica │
+│ 6   │ SepalWidth │ 3.0   │ virginica │
+
 ```
 
 `melt` is an alternative function to reshape from wide to long format. It is based on `stack`, but it prefers specification of the id columns as:
 
-```julia
-d = melt(iris, :Species)
-```
+```jldoctest
+julia> using DataFrames, CSV
 
-All other columns are assumed to be measured variables (they are stacked).
```

Comment:
Why remove these two sentences?

Reply:
I've also removed the example they describe, since it doesn't work and I'm not sure what it was trying to achieve that isn't covered by the other examples